Intersect.Client.Framework/Database/GameDatabase.cs

```diff
@@ -26,17 +23,25 @@ public abstract partial class GameDatabase
 
         public bool StickyTarget { get; set; }
 
-        public bool FriendOverheadInfo;
+        public bool FriendOverheadInfo { get; set; }
+
+        public bool GuildMemberOverheadInfo { get; set; }
 
-        public bool GuildMemberOverheadInfo;
+        public bool MyOverheadInfo { get; set; }
 
-        public bool MyOverheadInfo;
+        public bool NpcOverheadInfo { get; set; }
+
+        public bool PartyMemberOverheadInfo { get; set; }
+
+        public bool PlayerOverheadInfo { get; set; }
+
+        public bool HideOthersOnWindowOpen { get; set; }
```

Comment:
Why was this moved? It made reading this diff harder than necessary.

Reply:
My bad, at some point i meant to re-organize these variables according to their categories with comments, let me know if i should undo this one. as for the other properties, I've just added their missing `{ get; set; }`

Reply:
moved back `HideOthersOnWindowOpen`